psi4/src/read_options.cc

```diff
@@ -2026,6 +2026,8 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_int("VECS_CC3", 10);
         /*- Do collapse with last vector? -*/
         options.add_bool("COLLAPSE_WITH_LAST", true);
+        /*- Do collapse with last vector in CC3 iterations? -*/
+        options.add_bool("COLLAPSE_WITH_LAST_CC3", false);
```

Comment:
Would it be useful to describe the method scope of plain `COLLAPSE_WITH_LAST` above or note whether CC3 needs both or just the new one?

Reply:
Yes, better to document better - while it is in my mind again to remember and record!  I just added some verbosity.